lib/lit/cloud_translation/providers/google.rb

```diff
@@ -0,0 +1,111 @@
+# frozen_string_literal: true
+
+require_relative 'base'
+require 'google/cloud/translate'
```

Comment:
I think It can be made more user friendly for situations when gem `google-cloud-translate` is not added to `Gemfile`

```ruby
begin
  require 'google/cloud/translate'
rescue LoadError
  raise 'You need to add "gem \'google-cloud-translate\'" to your Gemfile to support Google Cloud Translation'
end
```